### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by yashk0thari**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -162,3 +162,8 @@
   title: Code-In Experience
   img: himani.png
   author_name: Himani Yadav
+
+- url: https://betaking146970080.wordpress.com/2017/12/21/42/
+  title: Code is Life!!
+  img: Beta_KIng.jpg
```
Comment (anshumanv) [inline _data/blogs.yml:168]: Please upload the image for this blog.

**Review by anshumanv — CHANGES_REQUESTED** [review]
Check comments

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Resolve conflicts !!

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@Beta-King  Please follow `README.md` on where to add blog image, Please update this asap 👍

**Comment by yashk0thari** [comment]
i've done this in another PR which is merged, sorry i forgot to close this one.

**Pull request closed without merging**